plotly/tools.py

```diff
@@ -1503,35 +1503,53 @@ def _unconvert_from_RGB_255(colors):
             return un_rgb_colors
 
     @staticmethod
-    def _map_array2color(array, colormap, vmin, vmax):
+    def _map_face2color(face, colormap, vmin, vmax):
         """
-        Normalize values in array by vmin/vmax and return plotly color strings.
+        Normalize facecolor values by vmin/vmax and return rgb-color strings
 
-        This function takes an array of values along with a colormap and a
-        minimum (vmin) and maximum (vmax) range of possible z values for the
-        given parametrized surface. It returns an rgb color based on the
-        relative position of zval between vmin and vmax
+        This function takes a tuple color with elements between 0 and 1, along
```

Comment:
> distances don't need to be 0 < color < 1, right? Since you're standardizing them by vmin and vmax

Right again but they will all be converted to colors with elements between 0 and 1 anyways. I updated the string doc again. =)